### PR TITLE
Java backend

### PR DESCRIPTION
**Pull request by svrnm**

### CODE REVIEW

File: app/backend2/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-ENTRYPOINT ["dotnet", "backend2.dll"]
+WORKDIR /usr/src/app
+
+COPY --from=builder /usr/src/app/ .
+
+# CMD ["java", "-jar", "./build/libs/dice-0.0.1-SNAPSHOT.jar"]
+
+# For debug purpose:
+# Comment the CMD above and uncomment the lines below to add the OTel java agent 
+ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar /javaagent.jar
```
Comment (pavolloffay) [inline app/backend2/Dockerfile:18]: Shall we remove this and use the auto-inject from the operator?

File: app/backend3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build-env
```
Comment (pavolloffay) [inline app/backend3/Dockerfile:1]: so the dotnet stays and is moved to backed3?

Comment (svrnm) [reply app/backend3/Dockerfile:1]: exactly, but not used, it's just there if someone wants to expand on it later

File: app/backend2/Dockerfile (hunk 1 above)
Comment (svrnm) [reply app/backend2/Dockerfile:18]: ah, I forgot to comment that one out and uncomment the CMD above, this is only here for debugging purpose

**Pull request merged**